mealie/routes/admin/admin_invitations.py

```diff
@@ -0,0 +1,25 @@
+from fastapi import APIRouter
+
+from mealie.routes._base import BaseAdminController, controller
+from mealie.schema.household.invite_token import (
+    ReadInviteToken,
+)
+from mealie.schema.response.pagination import PaginationQuery
+
+router = APIRouter(prefix="/invitations")
+
+
+@controller(router)
+class AdminInvitationsController(BaseAdminController):
+    @router.get("", response_model=list[ReadInviteToken])
+    def get_all(self):
+        """Get all invite tokens"""
+        self.repos.group_id = None
+        self.repos.household_id = None
```

Comment:
By default admin controllers don't filter by group or household, so you can remove these lines